package.json

```diff
@@ -56,7 +56,7 @@
     "extend": "^3.0.1",
     "functional-red-black-tree": "^1.0.1",
     "google-gax": "^0.14.1",
-    "grpc": "^1.6.0",
+    "grpc": "1.7.1",
```

Comment:
Is there a way to specify >= 1.6.0, < 1.8?

Reply:
Not as far as I know. >= 1.7.0 to < 1.8.0 is the closest we got.